code/__HELPERS/time.dm

```diff
@@ -20,6 +20,10 @@
 /proc/SQLtime(timevar)
 	return time2text(timevar || world.timeofday, "YYYY-MM-DD hh:mm:ss")
 
+/proc/station_time(display_only = FALSE, wtime=world.time)
```

Comment:
Why is this a blank proc now? Since this proc is blank, this means it now does literally nothing.